plugin/v2/settable.go

```diff
@@ -0,0 +1,89 @@
+package v2
+
+import (
+	"fmt"
+	"strings"
+)
+
+type settable struct {
+	name  string
+	field string
+	value string
+}
+
+var (
+	allowedSettableFieldsEnv     = []string{"value"}
+	allowedSettableFieldsArgs    = []string{"value"}
+	allowedSettableFieldsDevices = []string{"path"}
+	allowedSettableFieldsMounts  = []string{"source"}
+)
+
+func newSettables(args []string) []settable {
+	settables := []settable{}
+	for _, arg := range args {
+		settables = append(settables, newSettable(arg))
+	}
+	return settables
+}
+
+func newSettable(arg string) settable {
+	var set settable
+	if i := strings.Index(arg, "="); i < 0 {
+		set.name = arg
+		set.value = "true"
```

Comment:
actually, let's not do anything, so we allow to set the env to `""`. no reason to error WDYT ?